docs/source/daml/reference/interfaces.rst

```diff
@@ -32,6 +35,28 @@ Interface Name
 - It's preceded by the keyword ``interface`` and followed by the keyword ``where``.
 - It must begin with a capital letter, like any other type name.
 
+Implicit abstract type
+----------------------
+
+- Whenever an interface is defined, an abstract type is defined with the same
```

Comment:
i wonder if `opaque` might be clearer to avoid confusion with abstract classes which feel a bit different. Don't feel strongly about this.

Reply:
Hm I like "opaque" but I'll keep "abstract" for consistency with the introduction "In Daml, an interface defines an abstract type…", because "opaque" doesn't quite work there